maestro-cli/src/main/java/maestro/cli/report/HtmlAITestSuiteReporter.kt

```diff
@@ -0,0 +1,165 @@
+package maestro.cli.report
+
+import kotlinx.html.a
+import kotlinx.html.body
+import kotlinx.html.button
+import kotlinx.html.div
+import kotlinx.html.h1
+import kotlinx.html.head
+import kotlinx.html.html
+import kotlinx.html.img
+import kotlinx.html.lang
+import kotlinx.html.main
+import kotlinx.html.meta
+import kotlinx.html.p
+import kotlinx.html.script
+import kotlinx.html.span
+import kotlinx.html.stream.appendHTML
+import kotlinx.html.style
+import kotlinx.html.title
+import kotlinx.html.unsafe
+import readResourceAsText
+import java.io.File
+
+// TODO(bartekpacia): Decide if AI output can be considered "test output", and therefore be present in e.g. JUnit report
```

Comment:
Not sure if I understand, are you suggesting to use AI for giving HTML output?

Reply:
Nope.

I suggest merging (1) _the default HTML report_ and (2) _the AI results HTML report_ into a one output HTML file, for consistency and easy of use by the user.

I will rewrite this comment to make this more understandable.

PS Tangentially, I believe we should really improve what our current HTML report looks like. I think it's ugly and doesn't provide much value. There are already some community PRs that aim to improve this: https://github.com/mobile-dev-inc/maestro/pull/1864

Reply:
Got it sounds good!